content/articles/How-to-build-a-location-based-app-using-Django-and-GeoDjango/index.md

```diff
@@ -0,0 +1,280 @@
+﻿In this tutorial, we will build a web-app that stores all the coordinates of hotels in a particular region. Whenever a user enters their coordinate, it will display all the hotels that are close to the user's coordinate.
+
+
+### Table of contents
+- [Prerequisite](#prerequisite)
+- [Objectives](#objectives)
+- [Setup](#setup)
+- [Database Connection](#database-connection)
+- [Importing Dummy Data](#importing-dummy-data)
+- [Creating Homepage](#creating-homepage)
+- [Conclusion](#conclusion)
+- [Further Reading](#further-reading)
+
+### Prerequisite
+- GeoDjango 
+- Django
+- PostgreSQL
+- PostGIS extension
+- Overpass Turbo.
+- PostgreSQL
+- Pycharm or any text editor
+
+Let us look at the tools that we will use to build our app:
+**GeoDjango:** is a Django module that is used for creating geographic applications, manage spatial databases in Python and is integrated with Django. It can also work as a standalone framework.
+**Django:** is a python framework that uses the model–template–views (MTV) architectural pattern and is used for building web-apps.
+**PostgreSQL:** is a Relational Database Management System that is free and open-source, it focuses on maintaining extensibility and SQL compliance.
+**PostGIS:** is a PostgreSQL addon that manages spatial data. 
+**Overpass Turbo:** is an OpenStreetMap web-based tool for analysing large data.
+
+### Objectives
+The aim of this tutorial is to teach reader, how to:
+- Build a gis application from scratch
+- Create a spatial database and use spatial features
+- Implement geolocation features
+
+### Setup
+We will create our virtual environment and also install the prerequisites. Let's type the following codes in our terminal so that we can create and activate the virtual environment. 
+```
+virtualenv env 
+env\scripts\activate
+```
```

Comment:
Include `pip3 install virtualenv`. Add the command to install virtualenv

Reply:
It's done